analyzers/src/SonarAnalyzer.CSharp/Rules/InsecureEncryptionAlgorithm.cs

```diff
@@ -56,7 +58,9 @@ public sealed class InsecureEncryptionAlgorithm : InsecureEncryptionAlgorithmBas
         protected override string StringLiteralValue(ArgumentSyntax argument) =>
             ((LiteralExpressionSyntax)argument.Expression).Token.ValueText;
 
-        protected override Location Location(ObjectCreationExpressionSyntax objectCreation) =>
-            objectCreation.Type.GetLocation();
+        protected override Location Location(SyntaxNode objectCreation) =>
+            objectCreation is ObjectCreationExpressionSyntax objectCreationExpression
```

Comment:
Same here

Reply:
We agreed to do it in slightly different way, since there is no Type part in `ImplicitObjectCreationException`